clients/admin-ui/src/features/system/system.slice.ts

```diff
@@ -129,7 +129,7 @@ export const selectActiveClassifySystemFidesKey = createSelector(
 
 export const selectAllSystems = createSelector(
   systemApi.endpoints.getAllSystems.select(),
-  ({ data }) => data
+  ({ data }) => data || []
```

Comment:
we use a pattern of declaring the empty array as a const, see https://github.com/ethyca/fides/pull/1083/files#r972486065

Reply:
Oh, that's helpful and makes sense. I'll add that now